pkg/component/gardenercontrollermanager/gardener_controller_manager_suite_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gardenercontrollermanager_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestGardenerScheduler(t *testing.T) {
```

Comment:
```suggestion
func TestGardenerControllerManager(t *testing.T) {
```

Reply:
Nice catch.
Will be adapted in the next commit.

Reply:
This is not yet fixed

Reply:
Fixed with https://github.com/gardener/gardener/pull/8282/commits/4c6bc79e35b3d6938ded055ad3d09e74112f9f5d.